dir/base.go

```diff
@@ -93,12 +93,21 @@ func loadPath() {
 		UserConfigFS: NewUnionDirFS(
 			NewRootedFS(userConfig, nil),
 		),
+		SystemConfigFS: NewUnionDirFS(
+			NewRootedFS(userConfig, nil),
+		),
```

Comment:
Why do we need `SystemConfigFS` and `UserConfigFS` when we already have `ConfigFS`? I am assuming because `ConfigFS` tries to tries to read the user dir first then the system dir? If so, it raises the question why do we need `fs.go`? Can we get rid of it and the complications that come with it and rewrite it as simple utility methods which return paths based on the config we define(in spec)

Reply:
The reason that I uses io/fs package is `./plugin/manager/manager.go` uses io/fs to WalkDir the directory to list all available plugins. According to https://github.com/notaryproject/notation/blob/main/specs/directory.md, if both system and user level have plugins, it should walk through both of them to find plugins. The UnionDirFS in fs.go provides a fs.FS to read multiple directories, so the [manager.go](https://github.com/notaryproject/notation-go/blob/main/plugin/manager/manager.go#L91) can use it. 

The UnionDirFS can read multiple directories with priority, however, it can not distinguish user or system directory, so I add 3 FS to cover different use cases.
```
                // need to walk through both user and system level
		ConfigFS: NewUnionDirFS(
			NewRootedFS(userConfig, nil),
			NewRootedFS(systemConfig, nil),
		),
                // write to user level
		UserConfigFS: NewUnionDirFS(
			NewRootedFS(userConfig, nil),
		),
                // write to system level
		SystemConfigFS: NewUnionDirFS(
			NewRootedFS(userConfig, nil),
		),
```

Reply:
Removed `UserConfigFS` and `SystemConfigFS`, please review it again.